packages/mobile/src/send/saga.ts

```diff
@@ -56,10 +57,16 @@ export async function getSendTxGas(
 export async function getSendFee(
   account: string,
   currency: CURRENCY_ENUM,
-  params: BasicTokenTransfer
+  params: BasicTokenTransfer,
+  includeDekFee: boolean = false
 ) {
   try {
-    const gas = await getSendTxGas(account, currency, params)
+    let gas = await getSendTxGas(account, currency, params)
+    if (includeDekFee) {
+      const dekGas = await getRegisterDekTxGas(account, currency)
+      gas = gas.plus(dekGas)
+    }
+
```

Comment:
Wondering, how long does it take to call `getRegisterDekTxGas`?

Reply:
It seems to be quick, no noticeable change in fee calculation time. The `getRegisterDekTxGas` was modeled closely after our other gas estimation methods.